README.md

```diff
@@ -703,6 +703,15 @@ Users can perform a bulk upload by using the upload-file command on a target fol
 ardrive upload-file --local-file-path /path/to/folder  --parent-folder-id "9af694f6-4cfc-4eee-88a8-1b02704760c0" -w /path/to/wallet.json
 ```
 
+### Uploading files using a glob pattern (BETA)
```

Comment:
Thanks for adding this section! You'll also need to add a link to it in the table of contents above. Since the heading is rather long, use an anchor with an id so that you can easily link to it from the ToC, e.g. like the above:
```
### Uploading a Folder with Files (Bulk Upload)<a id="bulk-upload"></a>
```